src/bloomd/networking.c

```diff
@@ -319,6 +361,15 @@ int init_networking(bloom_config *config, bloom_filtmgr *mgr, bloom_networking *
         return 1;
     }
 
+    // Setup the unix listener
+    if (config->unix_socket != "") {
```

Comment:
Comparing against a string literal is unspecified behavior, better to use strncmp instead.

Reply:
or it can be just strlen() != 0, as it's testing only empty value. I will push

Reply:
Although, it might be better to be consistent with https://github.com/Applifier/bloomd/pull/1/files#diff-905ec4b139b45bd7f3af99fedffbad74R19